src/desktop/apps/collect/client.js

```diff
@@ -10,6 +10,7 @@ buildClientApp({
   context: {
     user: sd.CURRENT_USER,
     mediator,
+    COLLECTION_HUBS: sd.COLLECTION_HUBS,
```

Comment:
We need to pass the A/B test over on both the client as well as the server. We can do this by injecting ENV var into context which we then pluck out via `SystemContext` in reaction.